build.gradle

```diff
@@ -4,7 +4,8 @@ plugins {
     id 'signing'
     id 'com.gorylenko.gradle-git-properties' version '2.3.2' apply false
     id 'com.github.ben-manes.versions' version '0.38.0'
-    id 'net.researchgate.release' version '2.8.1'
+    // TODO: Warning for Gradle 9 by https://github.com/researchgate/gradle-release/issues/379
+    id 'net.researchgate.release' version '3.0.2'
```

Comment:
Note... this is currently the only thing throw a warning, this not affect the process but for Gradle 9 this going to break the release tasks... this was reported in https://github.com/researchgate/gradle-release/issues/379 currently with not good responses (?